bot/utils/database-helper.js

```diff
@@ -51,4 +44,40 @@ module.exports.openDatabase = () => {
  */
 module.exports.getDatabase = () => {
     return database;
-};
\ No newline at end of file
+};
+
+/**
+ * Run all database migrations so that we reach the requested version.
+ * @param {string|undefined} version Which version to migrate to. If undefined, the latest version is chosen.
+ * @param {boolean} allowDown If true, allow downgrading database version. Otherwise (default), we throw an error.
+ */
+module.exports.doDatabaseMigrations = async (version = undefined, allowDown = false) => {
+    if (!version) {
+        version = migrations.latestMigration();
+    }
+
+    function exec(statements) {
+        return new Promise((resolve, reject) => {
+            module.exports.getDatabase().exec(statements, (err) => {
+                if (err) {
+                    Logger.log("Exec in database failed : " + statements, "err");
+                    reject(err);
+                }
+                resolve();
+            });
+        });
+    }
+
+    const diffList = migrations.listDiff(await migrations.current(), version);
+    for (let diff of diffList) {
+        if (diff[1] === "down" && (allowDown === false || allowDown === undefined)) {
+            throw new Error("May not downgrade without allowDown set to true.");
+        } else {
+            const statements = require(__dirname + "/../migrations/" + diff[0])[diff[1]]();
+            await exec(statements);
+        }
+    }
+
+    //Write the new current version to the database
+    await exec(`INSERT OR REPLACE INTO _Migrations(name, value) VALUES('version', '${version}');`);
```

Comment:
This is kind of ugly, but there is no easy CRUD for this operation as far as I could tell.
Maybe it should be made, because we have more places that have key-value stores like this (such as BotSettings).